00-open-a-shapefile.Rmd

```diff
@@ -159,31 +155,26 @@ aoiBoundary <- readOGR("boundaryFiles/HARV", "HarClip_UTMZ18")
 ```
 
 After loading a new data set we also want to view the attributes of the layer.
-This can be done simple by entering the file name.
+This can be done simple by entering the file name. Alternatively, we can use the `attributes()` function.
```

Comment:
I'm fine with either.

The line along the side is achieved with this syntax

    > text here
    > text here
    > more text here

It's nice because it allows us to use bullets, etc.
The notice highlight can only highlight ONE paragraph of text. 

Let's make a rule and stick with it. i haven't been consistent. I'll go with whatever you think works best!